--- ghostpdl-1.54/pl/plmain.c.orig	2012-11-19 00:05:12.958343465 +0100
+++ ghostpdl-1.54/pl/plmain.c	2012-11-19 00:05:55.211675307 +0100
@@ -266,7 +266,7 @@
     /* Create PDL instances, etc */
     if (pl_main_universe_init(&universe, err_buf, mem, pdl_implementation,
 			      pjl_instance, &inst, &pl_pre_finish_page, &pl_post_finish_page) < 0) {
-	errprintf(err_buf);
+	errprintf("%s", err_buf);
 	return -1;
     }
 
@@ -384,7 +384,7 @@
 							pl_select_implementation(pjl_instance, &inst, r),
 							&inst, (gs_param_list *)&params);
 		if ( curr_instance == NULL ) {
-		    dprintf(err_buf);
+		    dputs(err_buf);
 		    return -1;
 		}
 
@@ -447,7 +447,7 @@
 
     /* Dnit PDLs */
     if (pl_main_universe_dnit(&universe, err_buf)) {
-	dprintf(err_buf);
+	dputs(err_buf);
 	return -1;
     }
     /* dnit pjl */
--- ghostpdl-1.54/pxl/pxtop.c~	2008-08-02 23:09:12.000000000 +0200
+++ ghostpdl-1.54/pxl/pxtop.c	2012-11-19 00:18:39.224981686 +0100
@@ -515,7 +515,7 @@
 	   code, st, pxs)) >= 0
 	)
 	  { if ( (report & eBackChannel) || force_to_cout )
-	      errprintf(message);
+	      errprintf("%s", message);
 	    if ( report & eErrorPage )
 	      y = px_error_page_show(message, y, pxs);
 	  }
